RELEASE.md

```diff
@@ -0,0 +1,54 @@
+# Releases
+Releases will be created in this repository for all official builds of `aurae` and associated projects.
+
+## Versioning
+Stable versions will follow a `vX.Y.Z` format. Versions will be consistent across everything being released.
```

Comment:
as above: crate versions or proto API versions?

Reply:
I remember we had a discussion on versioning a while back where it was suggested that v0 (proto or otherwise) would be considered unstable and any update would be open to containing a breaking change.

Reply:
Lots of good info starting here in [discord](https://discord.com/channels/1026578122969206796/1026613727157567518/1034172135125549136). Im going through this now.

Reply:
I dont quite have an answer yet for versioning when you include the factors you've mentioned. I'm still working on putting together something that makes sense.

Reply:
I'm not sure I know enough to be able to answer these questions. I wonder if we might want to put something like "Anything released during an alpha phase is only meant to work with the same version" until we have a little bit more practice/process defined here.